framework/src/play/mvc/Router.java

```diff
@@ -614,6 +622,26 @@ public static ActionDefinition reverse(String action, Map<String, Object> args)
         return matchingRoutes;
     }
 
+  /**
+   * Prioritize action routes based on active locale and Play.langs properties. Active lang has highest priority, then prioritized according to Play.langs order.
+   *
+   * @param matchingRoutes
+   */
+  private static void prioritizeActionRoutesBasedOnActiveLocale(List<Router.ActionRoute> matchingRoutes) {
+        if(matchingRoutes.size()==0) return;
+        final String locale = Lang.get();
+        if(StringUtils.isEmpty(locale)) return;
+        matchingRoutes.sort(new Comparator<ActionRoute>() {
```

Comment:
Jenkins cannot compile the code 
```
    [javac]         matchingRoutes.sort(new Comparator<ActionRoute>() {
    [javac]                       ^
    [javac]   symbol:   method sort(<anonymous Comparator<ActionRoute>>)
    [javac]   location: variable matchingRoutes of type List<ActionRoute>
```
Seems like some method are missing as you use the interface Comparator